### PR TITLE
doc terms and services changed

### PR DESCRIPTION
**Pull request by Harshwardhan18**

### CODE REVIEW

File: src/app/shared/models/constants/transalation.json.ts [hunk 1]
```diff
@@ -1,1847 +1,2008 @@
 export const HindiLang = [
   {
-    "text": "(Effective 2nd September 2018. Last Revised 13th August 2019)",
-    "translation": "(2 सितम्बर 2018 से लागु. अंतिम संशोधित तिथि 13 अगस्त 2019)"
+    text: '(Effective 2nd September 2018. Last Revised 13th August 2019)',
```
Comment (Manzurkds) [inline src/app/shared/models/constants/transalation.json.ts]: @Harshwardhan18 Can you disable the formatting here? This is a json data, the keys and values ideally need to be in double quotes

**Review by Manzurkds — CHANGES_REQUESTED** [review]

File: src/app/shared/services/translation.service.ts [hunk 2]
```diff
@@ -41,6 +41,7 @@ constructor(
             this.setIndex();
         } else {
             const lang = HindiLang;
+            lang.sort(this.sortLang);
```
Comment (Manzurkds) [inline src/app/shared/services/translation.service.ts:44]: @Harshwardhan18 You didn't use the script for sorting?

**Review by Manzurkds — CHANGES_REQUESTED** [review]

File: src/app/shared/services/translation.service.ts (hunk 2 above)
Comment (Harshwardhan18) [reply src/app/shared/services/translation.service.ts:44]: @Manzurkds I used the script but somehow the sorted JSON file was not working I tried this and it worked so added and removed the need to sort the script from an outer source.

Comment (Manzurkds) [reply src/app/shared/services/translation.service.ts:44]: @Harshwardhan18 Don't add it here, create a sortTranslation.js file in the repo which will import the json file, sort the content, and then write the sorted content on the file.
Add a script tag in package.json to run this file like 
`sort-translations: node sortTranslations.js`
Run this script tag before doing a build

Comment (Manzurkds) [reply src/app/shared/services/translation.service.ts:44]: Adding it here means, we will be sorting the translations again and again on every load, which doesn't make any sense.

**Review by Manzurkds — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -4,10 +4,11 @@
   "scripts": {
     "ng": "ng",
     "start": "node server.js",
+    "sort-translations": "node sortTranslation.js",
     "serve:staging": "ng serve --configuration=dev-staging",
     "serve:prod": "ng serve --configuration=dev-production",
     "build": "ng build --configuration=production",
-    "build:production": "ng build --configuration=production",
+    "build:production": "sort-translations && ng build --configuration=production",
     "build:staging": "ng build --configuration=staging",
```
Comment (Manzurkds) [inline package.json:12]: @Harshwardhan18 Why is this not for staging?

File: sortTranslation.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+// import { HindiLang } from './src/app/shared/models/constants/transalation.json';
+
+const fs = require('fs');
+const fileName = './src/app/shared/models/constants/transalation.json'
```
Comment (Manzurkds) [inline sortTranslation.js:4]: @Harshwardhan18 Fix the spelling of translation

**Review by Manzurkds — CHANGES_REQUESTED** [review]

**Review by Manzurkds — APPROVED** [review]

**Pull request merged**